ansible_collections/arista/avd/plugins/filter/convert_dicts.py

```diff
@@ -8,7 +8,7 @@
 import os
 
 
-def convert_dicts(dictionary, primary_key="name"):
+def convert_dicts(dictionary, primary_key="name", secondary_key=None):
     """
```

Comment:
Please update filter docstring, Markdown documentation for filters and ansible-test coverage.

Reply:
what is to be done in ansible-test coverage?

Reply:
You are adding a new functionality to the filter, so the unit tests should be added to test this out with all relevant variations.

Reply:
okay, I have updated docstring and Markdown documentation, getting a CI check failed for mkdoc.

Reply:
added test cases

Reply:
looks good. Thank you!